src/main/groovy/life/qbic/datamodel/datasets/MaxQuantRunResult.groovy

```diff
@@ -54,23 +50,19 @@ final class MaxQuantRunResult {
 
     private final SampleIds sampleIds
 
-    private final Summary summary
-
-    MaxQuantRunResult(AllPeptides allPeptides, Evidence evidence, ExperimentalDesignTemplate experimentalDesignTemplate, Parameters parameters, Peptides peptides, ProteinGroups proteinGroups, RunParameters runParameters, SampleIds sampleIds, Summary summary) {
+    MaxQuantRunResult(AllPeptides allPeptides, Evidence evidence, Parameters parameters, Peptides peptides, ProteinGroups proteinGroups, RunParameters runParameters, SampleIds sampleIds) {
```

Comment:
If we remove those values from the constructor we need to either create a new major release now or make the fields and constructor deprecated and then create a new constructor so that both are available.

Reply:
agree, I changed it!